### DISCUSSION TITLE
Failed to start compilation:

### DISCUSSION THREAD

**Discussion by RamiAidoui** [cpeditor/cpeditor#1279] [Q&A]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

[09:38:47] [Compiler] [Failed to start compilation: Failed to start the compiler. Please check [Preferences->Language->C++->C++ Commands->C++ Compile Command] or add the compiler in the PATH environment variable.]

### Expected Behavior

.

### Steps To Reproduce

.

### CP Editor Version

6.11.1

### OS

Windows 10

### Are you willing to contribute to this?

- [ ] I'm ready to fix this (and will open a PR soon)
- [ ] I'd like to have a try (with the help of the maintainers)
- [X] No, thanks

### Anything else?

;

**Comment by ouuan** [comment]
So have you installed a C++ compiler?

**Reply by WarLord0401** [reply]
I did. It works on Visual Studio code. but here it shows same error as the OP :(
